SU-HSDO/suhumsci
# shs-81: bug fix for traditional layered slider at large screen size

## Summary
This fixes a bug with the layered hero slider in a collection of two at large screen sizes in the traditional theme.

## Steps to Test
1. Create a new page. In the main region add a collection of two. Both of these items should be the Hero Layered Slider option.
2. Make sure the theme is Traditional. 
3. View the page at greater than `1500px`. Notice that the arrows for the slider are not off the box. 
4. Check to make sure this doesn't break a single layered hero slider (non collection)
5. Check to make sure this doesn't affect the colorful theme (this should only affect traditional - there's a separate bug for colorful).

## Screenshots
![Screen Shot 2022-08-15 at 12 43 25 PM](https://user-images.githubusercontent.com/8405274/184678530-1f151e04-8fb6-4cd0-bdcf-5eabcd0085ca.png)
